### PR TITLE
Update io.github.julian_hochhaus.LG4X_V2.yaml

### PR DESCRIPTION
**Pull request by Julian-Hochhaus**
Preparing Release of version 2.1.5 to flatpak

### CODE REVIEW

**Comment by kevinsmia1939** [comment]
bot, build io.github.julian_hochhaus.LG4X_V2

**Comment by Julian-Hochhaus** [comment]
@kevinsmia1939 do you know, how to fix or what's wrong? 

Did I at least set up everything more or less correctly? :D

**Comment by kevinsmia1939** [comment]
The git commit does not match the tag.

**Comment by kevinsmia1939** [comment]
I add a new PR here https://github.com/flathub/io.github.julian_hochhaus.LG4X_V2/pull/3

**Comment by Julian-Hochhaus** [comment]
So to summarize: To push a new version to flathub, the steps are:

1. update [metainfo file in main code repo](https://github.com/Julian-Hochhaus/LG4X-V2/blob/4afde50a8b7732d01e951e348806ec90a4ed0965/data/io.github.julian_hochhaus.LG4X_V2.metainfo.xml)
2. Fork [flathub-LG4X-V2-repo](https://github.com/flathub/io.github.julian_hochhaus.LG4X_V2)
3. update tag **and** corresponding commit in [LG4X-V2.yaml](https://github.com/flathub/io.github.julian_hochhaus.LG4X_V2/blob/master/io.github.julian_hochhaus.LG4X_V2.yaml)

In addition, I see that you are always adding`.patch` files. Are these needed and whats their content for? What do I need to include there?

File: io.github.julian_hochhaus.LG4X_V2.yaml [hunk 1]
```diff
@@ -82,7 +82,7 @@ modules:
       - type: git
         url: https://github.com/Julian-Hochhaus/LG4X-V2
         commit: 70f37b1ce388c4fcd65a02ae6302f1ae032ee760
-        tag: v2.1.0
+        tag: 2.1.5
```
Comment (hfiguiere) [inline io.github.julian_hochhaus.LG4X_V2.yaml:85]: you can't change a tag without changing the commit.

Comment (Julian-Hochhaus) [reply io.github.julian_hochhaus.LG4X_V2.yaml:85]: Yep, I already learned, it's my first time working with flathub, do you have a comprehensive starter guide on how to publish a new release of a piece of software on flathub?

Actually, all problems were already fixed in another PR by @kevinsmia1939

**Pull request closed without merging**